cmd/imagec/imagec.go

```diff
@@ -46,6 +46,9 @@ import (
 	"github.com/pkg/profile"
 )
 
+// ImageLayerIndex is the index of the child most layer in the manifest
+const ImageLayerIndex = 0
+
```

Comment:
[minor] as far as I see we only use this const once so I suggest simply using 0 in the code below :)